### PR TITLE
Refining GROMACS script

### PR DESCRIPTION
**Pull request by daico007**
WIP - Refining GROMACS workflow scripts (input conditions + `project.py`)

### CODE REVIEW

File: reproducibility_project/src/engines/gromacs/project.py [hunk 1]
```diff
@@ -19,171 +16,138 @@ def __init__(self):
...
+            "fname": "nvt.mdp",
+            "template": f"{mdp_abs_path}/nvt_template.mdp.jinja",
+            "data": {"temp": job.sp.temperature},
+        },
+        "npt": {
+            "fname": "npt.mdp",
+            "template": f"{mdp_abs_path}/npt_template.mdp.jinja",
+            "data": {
+                "temp": job.sp.temperature,
+                "refp": job.sp.pressure * 0.01,  # convert from kPa to bar
```
Comment (justinGilmer) [inline reproducibility_project/src/engines/gromacs/project.py:67]: Lets leverage unyt here if we can

```py3
import unyt as u
press = job.sp.pressure * u.kPa
                "refp": press.to_value('bar'),  # convert from kPa to bar
```

**Review by justinGilmer — CHANGES_REQUESTED** [review]
Small changes, but this looks good!

File: reproducibility_project/src/engines/gromacs/project.py [hunk 2]
```diff
@@ -19,171 +16,174 @@ def __init__(self):
...
+        ff_name = "benzene_trappe-ua_like.xml"
+        ff_path = (
+            str(os.path.dirname(os.path.abspath(xmls.__file__))) + "/" + ff_name
+        )
+        system[0].save(
+            filename="init.top", forcefield_files=ff_path, overwrite=True
+        )
+
+    # Modify mdp files according to job statepoint parameters
+    import unyt as u
```
Comment (justinGilmer) [inline reproducibility_project/src/engines/gromacs/project.py:65]: Lets move this to the top of the file with the other imports.

**Review by justinGilmer — CHANGES_REQUESTED** [review]
Small change, then LGTM!

File: reproducibility_project/src/engines/gromacs/project.py (hunk 2 above)
Comment (daico007) [reply reproducibility_project/src/engines/gromacs/project.py]: Fixed

**Review by justinGilmer — APPROVED** [review]
Once we change the center of mass motion removal back from Angular to linear, this is good to go.

**Pull request merged**